source_path: "resources/test.avi"
fps: 30
